### PR TITLE
feat(lightning): sync stored ldk txs with activity list

### PR DESCRIPTION
**Pull request by Jasonvdb**
### Description

- Continues to show incoming tx animation even when invoice might not be tracked.
- Syncs stored claimed and sent txs from react-native-ldk with activity list.
- Removes pending invoice tracking. That functionality is now in react-native-ldk.
- Shows incoming payment animation regardless of whether or not the pending is tracked.
- Slashpay now checks stored claimed payments when refreshing the drive's invoice

### Linked Issues/Tasks
https://github.com/synonymdev/bitkit/issues/1007

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [x] No test

### QA Notes

Hoping incoming animation shows up but if it doesn't at least tx history won't be missing

### CODE REVIEW

File: src/utils/lightning/index.ts [hunk 1]
```diff
@@ -1500,3 +1494,64 @@ export const getLightningBalance = ({
...
+		}
+	});
+
+	dispatch({
+		type: actions.UPDATE_ACTIVITY_ITEMS,
+		payload: items,
+	});
+
+	return ok('Stored lightning transactions synced with activity list.');
+};
```
Comment (coreyphillips) [inline src/utils/lightning/index.ts]: To keep all dispatch actions together the `syncLightningTxsWithActivityList` method should be moved to `src/store/actions/lightning.ts`

**Review by coreyphillips — CHANGES_REQUESTED** [review]

File: src/utils/lightning/index.ts (hunk 1 above)
Comment (Jasonvdb) [reply src/utils/lightning/index.ts]: Fixed here https://github.com/synonymdev/bitkit/pull/1171/commits/230ef2d0f861d49bd7eb8e415571717bb180c81d

**Review by coreyphillips — DISMISSED** [review]

**Comment by coreyphillips** [comment]
Not sure why the tests are failing.

**Review by coreyphillips — APPROVED** [review]

**Comment by Jasonvdb** [comment]
> Channel seems to remain as pending in Bitkit

Ah thanks might have been a race condition, channel was confirmed when I was testing receiving

**Pull request merged**